wagtail/images/jinja2tags.py

```diff
@@ -24,7 +29,7 @@ def image(image, filterspec, **attrs):
         return rendition
 
 
-def srcset_image(image, filterspec, **attrs):
+def srcset_image(image, filterspec, preserve_svg=False, **attrs):
```

Comment:
The `preserve_svg` parameter can be removed now:
```suggestion
def srcset_image(image, filterspec, **attrs):
```